lib/manager/gomod/artifacts.spec.ts

```diff
@@ -79,13 +74,15 @@ describe('manager/gomod/artifacts', () => {
     ).toBeNull();
     expect(execSnapshots).toMatchSnapshot();
   });
+
   it('returns null if unchanged', async () => {
-    fs.readFile.mockResolvedValueOnce('Current go.sum' as any);
-    fs.readFile.mockResolvedValueOnce(null as any); // vendor modules filename
+    fs.readLocalFile.mockResolvedValueOnce('Current go.sum');
+    fs.readLocalFile.mockResolvedValueOnce(null as any); // vendor modules filename
```

Comment:
```suggestion
    fs.readLocalFile.mockResolvedValueOnce(null); // vendor modules filename
```

is the cast required? If not, please remove all those any casts.